subgraph/l2/networks.json

```diff
@@ -0,0 +1,10 @@
+{
+  "optimism": {
+    "RequestManager": {
+      "address": "0xe236d136aD9f6dB125e9ba95eDc1d1A19423Ff0F"
+    },
+    "FillManager": {
+      "address": "0x85267688CE7677D69d75F288Ff3064431030aF9F"
+    }
```

Comment:
Would it make sense to have these addresses as an environment variable inside `.env` or something similar? I think it would spare us updates in the repo. Instead we would need to only update the .env file on the hosting side. Do you think that is possible?

Reply:
Yes, the addresses will be stored in the network file, I am currently editing these.

Reply:
What do you mean? Will you edit them to use environment variables or should we leave them like this for the time being?